README.md

```diff
@@ -97,7 +97,7 @@ Troubleshooting
 
 If your issue is not addressed above, you can get in touch with us personally:
 
-  slack: http://slackin.siacoin.com (ping @taek, @nemo, or @jordan)
+  slack: http://siatalk.sia.tech (ping @taek, @nemo, or @jordan)
```

Comment:
preferably https://slackin.sia.tech

The slackin link will send you an invite. The slack url itself is siatalk.slack.com